src/Microsoft.FeatureManagement/FeatureManagementOptions.cs

```diff
@@ -13,5 +17,11 @@ public class FeatureManagementOptions
         /// If missing feature filters are not ignored an exception will be thrown when attempting to evaluate a feature that depends on a missing feature filter.
         /// </summary>
         public bool IgnoreMissingFeatureFilters { get; set; }
+
+        /// <summary>
+        /// Override default behaviour of a missing feature
+        /// If a feature flag exists with no configuration then this delegate will be invoked enabling you to capture the missing feature name and/or override the default behaviour.
+        /// </summary>
+        public Func<string, Task<bool>> OnMissingFeature { get; set; }
```

Comment:
We probably should let the Func take an `ILogger` too, so someone can log warning or error even if they don't want to change the default behavior. Thoughts?

Reply:
Yeah, it sounds like a good idea. It'll save people from having to create/register an `IConfigureOptions<FeatureManagementOptions>` to inject a logger.